client/src/templates/layouts/Main.js

```diff
@@ -1,8 +1,14 @@
 import styled from "styled-components";
 
+import { mq } from "constants/theme";
+
 const Main = styled.main`
   margin-left: 25px;
   margin-right: 25px;
+
+  @media only screen and (min-width: ${mq.tablet.wide.minWidth}) {
```

Comment:
Not sure if this is good since it'll affect all components inside of Main.. If there's no workaround I guess we could go through and update components that were affected

Reply:
But don't all components inside of main have to have a min-width of 100vh- (footer + navbar) to ensure that the footer is at the bottom if the content is short?

Reply:
Ah sorry I commented this while on mobile and made a mistake, I meant to write this comment for line 10

Reply:
I meant that adding `margin: 0` may affect other components that are within the `Main` component

Reply:
I don't think there is another way to remove the margin on the landing page? Since the Main component will always set a 25px if we don't remove it and our content is rendered in Main for every page

Reply:
You're right though, that might effect other pages as well, I just don't know how to fix it without changing the margin here